### DISCUSSION TITLE
Wondering what im doing wrong, would love to figure this out

### DISCUSSION THREAD

**Discussion by hackadelic** [mhdzumair/MediaFusion#382] [Q&A]
So i am trying to run my own mediafusion instance, on a vm on a dedi that i own, when i try to follow the docker-compose instructions i kind of get stuck, i normally cant get the prowler setup to run because:
WARN[0000] volume "docker-compose_prowlarr-config" already exists but was not created by Docker Compose. Use `external: true` to use an existing volume
i've tried removing all the volumes and it still seems to say that, i've reinstalled the vm like 10 times and always get that same thing, if i set volumes to external for config it just wont run at all and will sit waiting for prowler to become ready.

now i've gotten past this before somehow, but then i could never get a reverse proxy to reach the service, i really dont understand why i have everything else running on my server using traefik but i cannot for the life of me to get it to work, hence why i set it up on a seperate vm and i can't even get caddy to work, ive tried disabling the nginx service and pointing the reverse proxy at the :8000 service port for mediafusion.. i am pretty stuck and confused but would absolutely love to get this to work, if anyone has any tips would be awesome! ive tried configs in many different ways and im usually pretty good at troubleshooting things but this one has been seriously confusing, lol well any help is super appreciated and happy thanksgiving everyone (:

**Comment by mhdzumair** [comment]
hey, thanks for trying this out. I'm just reading and now it's closed. So, you figured this out?

**Comment by hackadelic** [comment]
I believe so, thanks i love what you're doing.. is there a discord or anything with discussion or anything for MediaFusion? im in the servers for things like comet and zurg and whatnot and i really enjoy learning and working with docker/kubernetes and what not

**Reply by hackadelic** [reply]
issue i have now is everything seems to be working, but every link that comes up in stremio after adding the addon and configuring i just get a "playback error: please try again" is there anything i can do about this?

**Comment by hackadelic** [comment]
just tried all the debrid sources i have premium on, RD, AD, TRB, and DL and they all give the playback error: please try again for every single result i get, it is showing me plenty of results but they all fail ): is there any way i can troubleshoot this? would definitely appreciate any help as i love mediafusion and since i have my own dedicated servers running it on them makes sense, as i usually get ratelimited on the elfhosted instance
